interface-elements-for-web/articles/grid/customization-dialog/filtering-page.md

```diff
@@ -4,6 +4,24 @@ author: Anna Kondratova
 legacyId: 118822
 ---
 # Filtering Page
-The "Filtering" page allows you to create and change the filter conditions. For each column, select the criteria operator and specify the operand value.
+The **Filtering** page allows you to create and change filter conditions. 
+
+For each column, you can select the criteria operator and specify the operand value.
+
+* Use ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-expandfilter.png) to expand a column filter condition.
+* Use ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-removefilter.png) or clear an operand value, to remove a column filter condition.
```

Comment:
"Use ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-removefilter.png) or clear an operand value, to remove a column filter condition" Explain.

Reply:
A column's filter can be removed in two ways: by clicking a RemoveFilter  button or by clearing an editor with filter condition.  Perhaps:
"Click ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-removefilter.png) or clear an operand value's editor to remove a column's filter."